File: Java/MathematicalPrograms/FindingTheCostOfAnItem.java
// Finding the total cost of the item if quantity and price is given
import java.util.Scanner;
public class FindingTheCostOfAnItem {
	public static void main(String[] args) {
		 Scanner myObj = new Scanner(System.in);
		 System.out.println("Enter the name of the Item");
		 String itemname = myObj.nextLine();
		 System.out.println("Enter the quantity of the Item. Enter in Kilograms");
		 int weight = myObj.nextInt();
		 System.out.println("Enter the base price of the Item");
		 int baseprice = myObj.nextInt();
		 int finalprice = baseprice*weight;
		 System.out.println("Total Price for the item " + itemname + " is Rs. " + finalprice);
	}
}

File: C++/MathematicalPrograms/FindingTheCostOfAnItem.cpp
//Finding the total cost of the item if quantity and price is given
#include <iostream>
#include <string>

using namespace std;

int main() {
	string itemname;
	int weight;
	int baseprice;
	int finalprice;
	cout << "Enter the name of the Item";
	getline(cin, itemname);
	cout << "Enter the quantity of the Item. Enter in Kilograms";
	cin >> weight;
	cout << "Enter the base price of the Item";
	cin >> baseprice;
	finalprice = baseprice*weight;
	cout << "Total Price for the item " << itemname << " is Rs. " << finalprice;
}